config/development.json

```diff
@@ -193,24 +192,7 @@
 		"upsell/concierge-session": true,
 		"upsell/nudge-component": true,
 		"woocommerce/extension-dashboard": true,
```

Comment:
`extension-dashboard` is also enabled everywhere but Desktop. Intent or oversight?